packages/@n8n/api-types/src/push/execution.ts

```diff
@@ -9,6 +9,7 @@ type ExecutionStarted = {
 		workflowId: string;
 		workflowName?: string;
 		retryOf?: string;
+		flattedRunData?: string;
```

Comment:
Should we make this non-optional? for full executions this will simply be `[{}]`. That way we can simplify the rest of the code a tiny bit.